src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -104,24 +90,23 @@ class DraftJSEditor extends React.Component<Props> {
 
         return (
             <div className={classes}>
-                <Label hideLabel={hideLabel} showOptionalText={!isRequired} text={label}>
-                    <Tooltip isShown={!!error} position="bottom-left" text={error ? error.message : ''} theme="error">
-                        {/* need div so tooltip can set aria-describedby */}
-                        <div>
-                            <Editor
-                                {...a11yProps}
-                                editorState={editorState}
-                                handleReturn={this.handleReturn}
-                                onBlur={handleBlur}
-                                onChange={handleChange}
-                                onFocus={onFocus}
-                                placeholder={placeholder}
-                                readOnly={isDisabled}
-                                stripPastedStyles
-                            />
-                        </div>
-                    </Tooltip>
-                </Label>
+                <Tooltip isShown={!!error} position="bottom-left" text={error ? error.message : ''} theme="error">
```

Comment:
An `aria-label` is not visible, so it's not a replacement for a `label` tag except in the case where the the `hideLabel` prop for this component causes it to be visually hidden.

The problem we are trying to solve is that `label` can only label a "real" form element, such as an `input`, and not the virtual form element created by DraftJS. An alternative is to use `aria-labelledby="some-span-id-123"` for the visual case, and either leave it and hide it or switch over to `aria-label="some string"` for the hidden case.

Also, removing the `label` prop is a breaking change that we probably don't want to manage right now, so it would be preferable to fix this using the same props.

https://github.com/facebook/draft-js/blob/85764327e49f94397a83c3431f5a03cc02a18104/src/component/base/DraftEditorProps.js#L89-L97